peps/pep-0771.rst

```diff
@@ -41,7 +41,26 @@ control can use ``package``. However, in practice, many users are unaware
 of the ``[recommended]`` syntax, placing the burden on them to know this for a
 typical installation. Having a way to have recommended dependencies be installed
 by default while providing a way for users to request a more minimal installation
-would satisfy this use case. The present PEP will describe a solution for this use case.
+would satisfy this use case, and the present PEP will describe a solution to this.
+
+Real-world examples of packages that encourage users to include extra
+dependencies by default include for example:
+
+* `astropy <https://www.astropy.org/>`_, a core package for astronomy, which
+  recommends that users install ``astropy[recommended]``. Some of the
+  dependencies in ``recommended``, such as `scipy <http://www.scipy.org>`_, are
+  used in a number of places in astropy for common
+  functionality and scipy should ideally be a required dependency, but there has
+  been reluctance to so as it is not a lightweight dependency and advanced users
+  who want minimal installs want the ability to opt out of it (however, as
+  described above, this places a burden on the majority of regular uses to
+  satisfy a use case for advanced users).
+* `fastapi <https://fastapi.tiangolo.com/>`_, a web framework, which recommends
+  that users install ``fastapi[standard]`` in order to include commonly needed
+  dependencies.
+* `tensorflow <https://www.tensorflow.org>`_, a machine learning platform, which
+  recommends that users install ``tensorflow[and-cuda]`` since CUDA support is
+  commonly needed by users.
```

Comment:
This section feels a bit wordy, and repeats the prose description from this section. Perhaps you could list the packages, and say that these fit the description in the paragraph above?

E.g. 

```rst
There are several packages that demonstrate this pattern by encouraging users to include extra dependencies by default, such as:

* (link to astropy): ``astropy[...]``
```

Reply:
Done